packages/shared/sierra-client/src/index.ts

```diff
@@ -1,7 +1,7 @@
-import type {AxiosInstance} from 'axios';
+import { APIResponse, errorResponse, ResponseStatus, successResponse, unhandledError } from '@weco/identity-common';
```

Comment:
Is there an auto-formatter you're using to fix these or just IntelliJ?

Reply:
The IntelliJ auto-formatter doesn't organise imports very well - or at all for that matter. The VSCode one is much better, so I ran that through them initially and then try and maintain that format by hand as time goes on.